src/app/static/src/app/components/adr/ADRIntegration.vue

```diff
@@ -1,7 +1,17 @@
 <template>
     <div v-if="loggedIn" class="mb-5">
         <adr-key></adr-key>
-        <select-dataset v-if="key"></select-dataset>
+        <div v-if="key">
+            <select-dataset></select-dataset>
+            <div id="adr-capacity">
+            <span class="font-weight-bold align-self-stretch"
+                  v-translate="'capacityAccessLevel'">
+           </span>
+                <span v-tooltip="handleUploadPermission(hasUploadPermission, true)">
```

Comment:
I think we could do with some extra whitespace - it's looking a bit cramped:

![Screenshot from 2021-03-29 18-08-36](https://user-images.githubusercontent.com/1724545/112873660-d57caf80-90b9-11eb-931a-1187abe0a63a.png)

Also: minor point but the formatting of this code makes it a bit hard to read